lib/rules/cli/4.1.yaml

```diff
@@ -1291,6 +1291,8 @@
 :oadm_release_info:
   :cmd: oc adm release info
   :options:
+    :image-for: --image-for=<value>
```

Comment:
@pdhamdhe ruby naming convention for names are usually `_` instead of `-`

Reply:
Done. Thanks